src/Draco.Compiler.Cli/Program.cs

```diff
@@ -103,7 +105,19 @@ private static void GenerateExe(FileInfo input, FileInfo output)
         var emitResult = compilation.Emit(dllStream);
         if (!emitResult.Success)
         {
-            foreach (var diag in emitResult.Diagnostics) Console.WriteLine(diag);
+            if (msbuildDiags) foreach (var diag in emitResult.Diagnostics)
+                    if (diag.Location.IsNone)
+                    {
+                        Console.WriteLine($"{diag.Severity.ToString().ToLower()} 000 : {diag.Message}");
+                    }
+                    else
+                    {
+                        Console.WriteLine($"""
+                        {diag.Location.SourceText.Path!.OriginalString}({diag.Location.Range!.Value.Start.Line + 1}, {diag.Location.Range!.Value.Start.Column + 1},
+                        {diag.Location.Range!.Value.End.Line + 1}, {diag.Location.Range!.Value.End.Column + 1}) : {diag.Severity.ToString().ToLower()} {diag.ErrorCode} : {diag.Message}
+                        """.ReplaceLineEndings(""));
+                    }
+            else foreach (var diag in emitResult.Diagnostics) Console.WriteLine(diag);
```

Comment:
Better, but let's keep the convention
don't omit braces if
 - The other branch needs it
 - The contained construct can also have blocks